cyences_app_for_splunk/default/data/ui/views/cs_oracle.xml

```diff
@@ -0,0 +1,59 @@
+<form version="1.1" theme="light">
+  <label>Oracle</label>
+  <fieldset submitButton="false">
+    <input type="time" token="tkn_time" searchWhenChanged="true">
+      <label>Time Range</label>
+      <default>
+        <earliest>-24h@h</earliest>
+        <latest>now</latest>
+      </default>
+    </input>
+    <input type="text" token="tkn_host" searchWhenChanged="true">
+      <label>Host</label>
+      <prefix>host="</prefix>
+      <suffix>"</suffix>
+      <default>*</default>
+      <initialValue>*</initialValue>
+    </input>
+    <input type="text" token="tkn_user" searchWhenChanged="true">
+      <label>User</label>
+      <default>*</default>
+      <prefix>user="</prefix>
+      <suffix>"</suffix>
+      <initialValue>*</initialValue>
+    </input>
+    <input type="text" token="tkn_action_name" searchWhenChanged="true">
+      <label>Action Name</label>
+      <default>*</default>
+      <prefix>vendor_action="</prefix>
+      <suffix>"</suffix>
+      <initialValue>*</initialValue>
+    </input>
+  </fieldset>
+  <row>
+    <panel>
+      <title>Audit Logs</title>
+      <table>
+        <search>
+          <query>`cs_oracle` sourcetype="oracle:audit:xml" $tkn_host$ $tkn_action_name$ vendor_action=*
```

Comment:
No need to add vendor_action=* 
$tkn_action_name$ token has default value for that.